prime-router/docker-compose.yml

```diff
@@ -25,4 +25,13 @@ services:
     image: mcr.microsoft.com/azure-storage/azurite
     ports:  
       - 10000:10000 
-      - 10001:10001
\ No newline at end of file
+      - 10001:10001
+
+#local SFTP server as a receive point
+  sftp:
+    image: atmoz/sftp
+    volumes:
+        - ./target/sftp:/home/foo/upload
+    ports:
```

Comment:
Thinking about it. I like that you are mapping the a target folder. It gets cleaned with `mvn clean`